example/scrashtest/spiders/dmoz_two.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+from urlparse import urljoin
+import json
+
+import scrapy
+from scrapy.contrib.linkextractors import LinkExtractor
+
+
+class DmozSpider(scrapy.Spider):
+    name = "js_spider"
+    start_urls = ['http://www.isjavascriptenabled.com/']
```

Comment:
well it's not really a test, just extra spider aside from existing dmoz spider, I used it for quick testing, can be removed no problem